### PR TITLE
offer pinterest-save extension

### PR DESCRIPTION
**Pull request by kevinlawler**
brave/browser-laptop#6366

compatible with base64 extension changes to muon
see https://github.com/brave/muon/commit/b346a92d64b5b99862fa182c0bc5d7d06306acac

Auditors:
@bridiver @jonathansampson

Test Plan:

linting

Auditors:

Test Plan:

Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [x] Added/updated tests for this change (for new code or code which already has tests).
- [x] Ran `git rebase -i` to squash commits (if needed).
- [x] Tagged reviewers and labelled the pull request [as needed](https://github.com/brave/browser-laptop/wiki/Pull-request-process).

Test Plan:

Reviewer Checklist:

Tests

- [ ] Adequate test coverage exists to prevent regressions
- [ ] Tests should be independent and work correctly when run individually or as a suite [ref](https://github.com/brave/browser-laptop/wiki/Code-Guidelines#test-dependencies)
- [ ] New files have MPL2 license header

### CODE REVIEW

File: app/extensions.js [hunk 1]
```diff
@@ -518,11 +522,17 @@ module.exports.init = () => {
     }
 
     if (getSetting(settings.HONEY_ENABLED)) {
-      registerComponent(config.honeyExtensionId)
+      registerComponent(config.honeyExtensionId, 'MIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQC79mayLyuCYY/dyT7Ycr1sVBp9yHrY4mnogVEgu+sDT6+/A121Na+aTw6mFLD6LHgbgHt4fnQ2V/QwcfBSXRTSkGpgNsZAjnYs4/XzZQYKGltWT93EP9zXN1kGbtzfkPGzTakquCfOjbKtbAQKWh8ppzqLhWcRUn9g/PhU99F29QIDAQAB')
```
Comment (bridiver) [inline app/extensions.js]: can we put this in config with the extension id?

Comment (kevinlawler) [inline app/extensions.js]: should be able to. let me see

Comment (kevinlawler) [inline app/extensions.js]: done

**Comment by NejcZdovc** [comment]
We need to pull this into 0.18, because we use the same Muon version now and 0.18 is failing because of that:

```
Error: `publicKeyString` is a required field
    at registerComponent (/Users/nejczdovc/Work/OpenSource/browser-laptop/app/extensions.js:446:24)
    at registerComponents (/Users/nejczdovc/Work/OpenSource/browser-laptop/app/extensions.js:472:7)
    at Object.module.exports.init (/Users/nejczdovc/Work/OpenSource/browser-laptop/app/extensions.js:531:3)
    at perWindowStateLoaded (/Users/nejczdovc/Work/OpenSource/browser-laptop/app/index.js:347:16)
    at Immediate.setImmediate (/Users/nejczdovc/Work/OpenSource/browser-laptop/app/index.js:337:24)
    at runCallback (timers.js:672:20)
    at tryOnImmediate (timers.js:645:5)
    at processImmediate [as _immediateCallback] (timers.js:617:5)
```

**Comment by bsclifton** [comment]
Setting back to 0.19.x.  I don't think we are able to use Muon 4.2.x in 0.18.x ☹️

**Pull request merged**